src/common/marketUtils.es6

```diff
@@ -12,4 +12,6 @@ const getMarketsSubmarkets = (active_symbols) => {
     return select_market_submarket
  }
 
-export default getMarketsSubmarkets;
\ No newline at end of file
+const getSortedMarkets = (markets) => markets.length > 1 ? Object.assign([], markets, {[2]: markets[3], [3]: markets[2]}) : markets;
```

Comment:
How is this function sorting the markets array? perhaps we could include the purpose of the function in either the name and/or parameters?

Reply:
because the returned array from the api is always vol indices first before commodities, that functions swapping the position between them both. as for if its norway clients where they only have volatility indices then it will stay as it is.